LayoutTests/platform/win/TestExpectations

```diff
@@ -895,7 +895,7 @@ fast/css/visited-link-mix-blend-mode.html [ Skip ]
 # Backdrop Filters do not work on Windows
 fast/css/will-change/will-change-creates-stacking-context-inline.html [ ImageOnlyFailure ] 
 
-# TODO CSS3_TEXT is disabled on Windows
+# TODO CSS Text 3 is not supported on Windows
 fast/css3-text/ [ Skip ]
```

Comment:
Please file a bug to get these unskipped on win/wincairo. I don't see why `text-align-last` and `text-justify` wouldn't be supported on Windows, since these shouldn't rely on platform specific things.

Reply:
I think I'll do an experiment: remove the skip, see how many tests in `fast/css3-text/` is failing on win/wincairo, add those failures into the test expectations.